### ISSUE TITLE
Accessibility. Group elements are not announced by screen reader.

### ISSUE THREAD

**Issue by rokalm** [ng-select/ng-select#1989]
**Describe the bug**
Group elements are not announced by screen reader.

**To Reproduce** 
1. Go to https://ng-select.github.io/ng-select#/grouping
2. Start narrator (in windows Win + Ctrl + Enter)
3. Open ng-select with groups
4. Navigate through options and notice that group is not announced

Expected: groups should be announced, see example here https://w3c.github.io/aria-practices/examples/listbox/listbox-grouped.html
